src/js/angular/sparql-editor/controllers.js

```diff
@@ -14,6 +14,7 @@ import 'services/ontotext-yasgui-web-component.service.js';
 import 'angular/externalsync/controllers';
 import {QueryType} from "../../../models/ontotext-yasgui/query-type";
 import {BeforeUpdateQueryResult, BeforeUpdateQueryResultStatus} from "../../../models/ontotext-yasgui/before-update-query-result";
+import Promise from "lodash/_Promise";
```

Comment:
Interesting. Where did you saw this?

Reply:
This is auto import of IDE. I will remove it.